js/contact-form.js

```diff
@@ -0,0 +1,142 @@
+const form = document.querySelector('.formidable');
+const username = document.getElementById('username');
+const email = document.getElementById('email');
+const textBox = document.getElementById('message');
+const inputArr = [username, email, textBox]
+
+
+const isRequired = (value) => value !== '';
+const checkEmail = function (input) {
+    const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+    let valid = false;
+    const emailVal = input.value.trim();
+    if (!isRequired(emailVal)) {
+        showError(input, 'Email cannot be blank.');
+
+    } else if (!re.test(emailVal)) {
+        showError(input, 'Email is not valid.');
+
+    } else if (input.value !== email.value.toLowerCase()) {
+        showError(input, 'Email must be in lowercase');
+
+    } else {
+        showSuccess(input);
+        valid = true;
+    }
+    return valid;
+}
+
+
+
+
+
+
+const message = function (input) {
+    const errorMessage = input.name.replace(/-p/, ' P');
+    return errorMessage.charAt(0).toUpperCase() + errorMessage.slice(1);
+}
+
+const showError = (input, message) => {
+    let formControl = input.parentElement;
+    formControl.classList = ' form-container error';
+    const small = formControl.querySelector('small')
+    small.innerText = message;
+}
+const showSuccess = (input) => {
+    let formControl = input.parentElement;
+    formControl.classList = ' form-container success'
+}
+
+
+
+
+const checkRequired = function (inputArr) {
+    inputArr.forEach(input => {
+        if (input.value.trim() === '') {
+            showError(input, `${message(input)} is required.`);
+            return
+        } else {
+            showSuccess(input)
+        }
+    })
+
+}
+
+
+const checkLength = function (input, min, max) {
+    if (input.value.trim().length < min) {
+        showError(input, `${message(input)} must contain ${min} characters`);
+    } else if (input.value.trim().length > max) {
+        showError(input, `${message(input)} must be less than ${max} characters`);
+    } else {
+        showSuccess(input);
+    }
+}
+
+
+
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    checkRequired(inputArr)
+    checkEmail(email);
+    checkLength(username, 5, 25)
+    checkLength(textBox, 5, 45)
+
+
+})
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
- [ ]  **[REQUIRED CHANGES]**

:point_right: **NOTE: Please, watch the video below to see your form in action**

- Please, kindly remove the unnecessary new lines added in your js file as it makes your codebase not clean. You should add a maximum of one new line per time between functions, and loops. event listeners. 
- According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md), Your form error message should show up near the submit button and not above the submit button. You can make it show just below the submit button.
Please, kindly fix this.
- A user cannot make a submission to your form. Please, kindly fix this also.
- Refer to the video below. :clinking_glasses: 

**YOUR FORM IN ACTION** :point_down: 

https://user-images.githubusercontent.com/59397159/168336872-aca1f7b9-fce7-4483-8a1b-afb5203ca88a.mp4